dashboard/app/assets/stylesheets/application.scss

```diff
@@ -1287,6 +1298,16 @@ a.download-video {
     text-transform: uppercase;
   }
 
+  #heading-banner-text {
+    color: $white;
+    margin-top: 40px;
+  }
+
+  #heading-banner-subtext {
+    color: $white;
+    margin-bottom: 115px;
+  }
```

Comment:
Oh, this was done for the pure-HAML header banner for signed-out homepage (https://github.com/code-dot-org/code-dot-org/pull/15197/files#diff-ed5055915737aa4392b75617e74d84ccR23) but I've since learned we don't want a version of the page for that case, so will remove in a moment.